examples/features/select-forwarder/service.yaml

```
@@ -7,9 +7,9 @@ metadata:
 spec:
   payload: ETHERNET
   matches:
```

Shouldn't this be the simpler:

```suggestion
  matches:
  - metadata:
      labels:
        my_forwarder_capability: true
```

Suppressing the empty fields?